harmonizer/Cargo.toml

```diff
@@ -25,6 +25,7 @@ apollo-federation-types = { version = "0.11.0", path = "../apollo-federation-typ
 deno_core = "0.200.0"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+tracing = "0.1.33"
```

Comment:
I am ... bearish ... on introducing `tracing` to this _just_ for the info logs below if we didn't already have something like that in this package, but i'll see if the team has any other suggestions.

Reply:
Ok let me know, happy to remove. Since it was in the bridge I added it but I see as well the argument against.